test/reifyhealth/specmonstah/core_test.cljc

```diff
@@ -693,17 +711,21 @@
            (sm/ent-relations db :t0)))))
 
 (deftest all-ent-relations
-  (is (= {:project   {:p0 {:created-by-id :u0
-                           :updated-by-id :u0
-                           :todo-list-ids #{:tl0 :tl1}}}
-          :user      {:u0 {}}
-          :todo-list {:tl0 {:created-by-id :u0
-                            :updated-by-id :u0}
-                      :tl1 {:created-by-id :u0
-                            :updated-by-id :u0}}}
-         (sm/all-ent-relations
-          (sm/build-ent-db {:schema td/schema}
-                           {:project [[:p0 {:refs {:todo-list-ids 2}}]]})))))
+  (let [db (sm/build-ent-db {:schema td/schema}
+                            {:project [[:p0 {:refs {:todo-list-ids 2}}]]})]
+    (is (= {:project   {:p0 {:created-by-id :u0
+                             :updated-by-id :u0
+                             :todo-list-ids #{:tl0 :tl1}}}
+            :user      {:u0 {}}
+            :todo-list {:tl0 {:created-by-id :u0
+                              :updated-by-id :u0}
+                        :tl1 {:created-by-id :u0
+                              :updated-by-id :u0}}}
+           (sm/all-ent-relations db)))
+    (is (= {:project   {:p0 {:created-by-id :u0
+                             :updated-by-id :u0
+                             :todo-list-ids #{:tl0 :tl1}}}}
+           (sm/all-ent-relations db [:p0])))))
```

Comment:
Might be nice to have an example with more than one ent so it's clear that's possible. Maybe `:p0` and `:tl0` so it's clear you can retrieve a portion of ents.